agent/src/Payload.php

```diff
@@ -8,9 +8,13 @@
 
 class Payload
 {
+    public const AGENT_PAYLOAD_VERSION = 'v1';
+
     public string $value = '';
 
-    public string $signature = '';
+    public string $payloadVersion = '';
```

Comment:
Because we are already in the payload class, I'd love if this were `$version` instead of `$payloadVersion`.

```php
$payload->payloadVersion;

// vs

$payload->version;
```